spec/javascript/containers/StateShowContainerTest.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import StateShowContainer from '../../../app/javascript/react/container/StateShowContainer'
+import ParkShowContainer from '../../../app/javascript/react/container/ParkShowContainer'
+import testHelper from '../testHelper'
+import {BrowserRouter} from "react-router-dom"
+
+
+describe('StateShowContainer', () => {
+  let wrapper
+  let stateShow
+  let parks
+
+  beforeEach(() => {
+    stateShow = [
```

Comment:
This is a nitpick, but since you don't need stateShow or parks in every test, please move these definitions into the relevant it block and remove the declarations in lines 10 and 11

```
it('should render a ParkShowContainer', () => {
   let stateShow = [
      ...
```